examples/shell/shell_common/cmd_ping.cpp

```diff
@@ -129,12 +129,60 @@ class PingArguments
 } gPingArguments;
 
 Protocols::Echo::EchoClient gEchoClient;
+Transport::AdminPairingTable gAdmins;
```

Comment:
Why we change gAdmins to globle?

Reply:
The `gAdmins` object requires its lifecylcle to be valid throughout the ping. Since we don't have a busy loop in the command handlers now we have to move it to global.